src/main/java/uk/gov/hmcts/juror/api/moj/repository/JurorPoolRepository.java

```diff
@@ -58,4 +58,9 @@ JurorPool findByOwnerAndJurorJurorNumberAndPoolPoolNumberAndIsActive(String owne
 
     @Query(value = "SELECT * from juror_mod.jsm_report_by_pool(:poolNumber)", nativeQuery = true)
     String getJsmReportByPool(@Param("poolNumber") String poolNumber) throws SQLException;
+
+    @Query(value = "SELECT * from juror_mod.jsm_report_by_court(:locCodes, :fromDate, :toDate)", nativeQuery = true)
+    String getJsmReportByCourt(@Param("locCodes") String poolNumber,
+                               @Param("fromDate") LocalDate fromDate,
+                               @Param("toDate") LocalDate toDate) throws SQLException;
```

Comment:
Do we need an explicit: throws SQLException;?

Reply:
Good to throw this so it can be dealt with by the client method